### PR TITLE
Fixed the column Sidebar and Cards on Networking Finding Tool. Changed Header title to Network...

### PR DESCRIPTION
**Pull request by mlipardo**
- Fix sidebar networking finding tool since Sidebar on react-wood-duck was updated.
- Change Header to **Network Finding Tool**.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -393,12 +393,6 @@ async-foreach@^0.1.3:
   version "0.1.3"
   resolved "https://registry.yarnpkg.com/async-foreach/-/async-foreach-0.1.3.tgz#36121f845c0578172de419a97dbeb1d16ec34542"
 
-async@2.4.1:
```
Comment (eckdanny-osi) [inline yarn.lock]: There should be no change to this file.

File: app/javascript/familyFinding/familyFinding.js [hunk 2]
```diff
@@ -29,31 +29,35 @@ export default class FamilyFinding extends React.Component {
     const cardProps = {
       cardHeaderText: 'Card',
       cardbgcolor: 'transparent',
-      columnLargeWidth: 9,
-      columnMediumWidth: 9,
-      offsetMediumValue: 3,
-      columnXsmallWidth: 9,
+      columnLargeWidth: 0,
```
Comment (eckdanny-osi) [inline app/javascript/familyFinding/familyFinding.js]: from RWD [source](https://github.com/ca-cwds/react-wood-duck/blob/master/src/Cards.js#L18), looks these defs produce the following css classes: `col-lg-0 col-md-0 col-md-offset-0 col-xs-0`.

I don't think these are BS3 classes. It there an issue upstream that is necessitating this hacky-feeling usage?

Comment (mlipardo) [reply app/javascript/familyFinding/familyFinding.js]: The card component has a default props so needs to overwrite, the card component may need to be refactored.

**Comment by mlipardo** [comment]
@edgibbs I did not update the version of react-wood-duck

**Pull request merged**